packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -226,7 +228,7 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
     const visibilityStyle: React.CSSProperties | undefined = this.props.hidden ? { visibility: 'hidden' } : undefined;
     // React.CSSProperties does not understand IRawStyle, so the inline animations will need to be cast as any for now.
     const content = (
-      <div ref={this._hostElement} className={this._classNames.container} style={visibilityStyle}>
+      <div ref={mergeRefs(this._hostElement, domRef)} className={this._classNames.container} style={visibilityStyle}>
```

Comment:
In cases like this it might be better to do this in the constructor and get rid of the merge refs call.
```
this._hostElement = props.domRef || React.createRef();
```
(and remove the old host element initialization)

Reply:
`domRef` could be a function (`(ref: T | null) => void`), which would break existing usage of the internal `_hostElement`